README.md

```diff
@@ -348,6 +348,13 @@ If contributing to the documentation, please edit [`README.md`](README.md) direc
 <!-- readme: collaborators,contributors -start --> 
 <table>
 <tr>
+    <td align="center">
```

Comment:
I was about to tell you you don't need to edit this because there's a bot to do that. Then I noticed the bot did that and applied the changes to your branch. 

Having the bot apply changes in forks is kind of annoying to contributors in general because now their local branches are suddenly not in sync with their own fork.

We need some rules in the workflow
https://github.com/alandefreitas/small/blob/9795e1f60eab809b200fc26ad3e7d05fef3cac42/.github/workflows/docs.yml#L3

to limit that to the master branch or something, as in
https://github.com/alandefreitas/small/blob/9795e1f60eab809b200fc26ad3e7d05fef3cac42/.github/workflows/build.yml#L3

We can open a new issue for that.

Reply:
Yeah, I guess you're right. Just to make sure I understand you, do we need to open an issue to constrain the bot from applying changes in arbitrary branches?  Like, constraining the bot to update those contributors' settings only in the master branch?

Reply:
> Just to make sure I understand you, do we need to open an issue to constrain the bot from applying changes in arbitrary branches?

Yes. Probably limit it to just 1) the master branch in 2) this repo. Unless we have an immediate solution, we can just open another issue.

> Like, constraining the bot to update those contributors' settings only in the master branch?

Only the master branch and only this repo.